airbyte-integrations/scripts/utils.sh

```diff
@@ -0,0 +1,37 @@
+die () {
+  echo "$1" 1>&2
+  exit 1
+}
+
+docker_build_tag () {
```

Comment:
I think we shouldn't have this method i.e: the calling context is easier to understand if I see `docker build -t <tag> -q .` instead of seeing a call to a method `docker_build_tag` whose definition I then have to look up